kolibri/plugins/epub_viewer/assets/src/views/SearchSideBar.vue

```diff
@@ -187,29 +186,20 @@
                 this.maxSearchResultsExceeded = true;
               }
               this.searchResults = searchResults.slice(0, MAX_SEARCH_RESULTS);
-
+              this.$emit('newSearchQuery', searchQuery);
+              this.searchIsLoading = false;
               // Wait for list to be updated
               this.$nextTick().then(() => {
-                if (this.markInstance) {
-                  this.markInstance.unmark({
```

Comment:
Since the node component now is always new, there is. no need to unmark anything, because nothing is marked yet.